crawler/src/main/java/com/movie/recommender/crawler/controller/CrawlerController.java

```diff
@@ -26,4 +28,16 @@ public ResponseEntity<String> triggerCrawl() {
             return ResponseEntity.internalServerError().body("An error occurred during the crawl process.");
         }
     }
+
+    @RabbitListener(queues = RabbitMQConfig.QUEUE_NAME)
+    public void listenToCrawlMessages(String message) {
```

Comment:
create a class `RabbitMQMessageListener` to handle RabbitMQ messages and move this method to the new class
place the class in the package `com.movie.recommender.crawler.messaging`